lib/puppet/pops/loader/module_loaders.rb

```diff
@@ -98,7 +98,8 @@ def initialize(parent_loader, loaders, module_name, path, loader_name, loadables
       @loaders = loaders
       @loadables = loadables
       unless (loadables - LOADABLE_KINDS).empty?
-        raise ArgumentError, 'given loadables are not of supported loadable kind'
+        #TRANSLATORS 'loadables' is a variable containing loadable modules
```

Comment:
Should this one also say, "and should not be translated" ? Looks like `loadables` is also a variable above

Reply:
done